libcxx/test/std/algorithms/alg.modifying.operations/alg.copy/copy.pass.cpp

```diff
@@ -15,23 +15,30 @@
 #include <algorithm>
 #include <cassert>
 
+#include "common.h"
 #include "test_macros.h"
 #include "test_iterators.h"
+#include "type_algorithms.h"
 
-class PaddedBase {
-public:
-  TEST_CONSTEXPR PaddedBase(std::int16_t a, std::int8_t b) : a_(a), b_(b) {}
-
-  std::int16_t a_;
-  std::int8_t b_;
-};
-
-class Derived : public PaddedBase {
-public:
-  TEST_CONSTEXPR Derived(std::int16_t a, std::int8_t b, std::int8_t c) : PaddedBase(a, b), c_(c) {}
+TEST_CONSTEXPR_CXX20 void test_padding() {
```

Comment:
Please don't put individual tests in functions. That makes the test just harder to read AFAICT, since it only adds additional code.

Reply:
I have reverted most test functions I created except for those in fill.pass.cpp. In fill.pass.cpp, I have combined multiple test functions with the same purpose into one function.